build/Dockerfile

```diff
@@ -66,6 +66,9 @@ RUN set -x \
 # Copy entrypoint script.
 COPY /docker-entrypoint.sh /
 
+# Set up healthcheck.
+HEALTHCHECK --interval=5m --start-period=15m --timeout=30s --retries=3 CMD $DSTA_HOME/version --check
```

Comment:
`CMD dst-server version --check`?

or `CMD $DSTA_HOME/bin/version.sh --check` but I don't like this one.

I think that there's nothing at `$DSTA_HOME/version`